test/controllers/tags_controller_test.rb

```diff
@@ -276,6 +285,9 @@ class TagsControllerTest < ActionController::TestCase
   # @param category [Category] category to create the tag in
   # @param tag_set [TagSet] tag set the tag belongs to
   def try_create_tag(tag_set, category, **opts)
+    # this is needed for draft deletion until we switch from request.referer
+    @request.set_header('HTTP_REFERER', create_tag_path(id: category.id))
```

Comment:
And this is only here because we rely on `request.referer` when creating and deleting drafts. I'd like to work on switching from that approach separately, but open to doing it "while we are at it" too.